### ISSUE TITLE
FIM Windows Agent - Case: Tags

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh-qa#1951]
## Description 

FIM tests have many flaws inconsistently. So our goal is to achieve consistent behavior.
The results are attached.

| Folder Path | Results |Date | By | Status| 
|--|--|--|--|--|
test_fim/test_files/test_tags  |  [ResultsTags.zip](https://github.com/wazuh/wazuh-qa/files/7259012/ResultsTags.zip) | 2021/09/29 | Daniela |  🔴

| Related to | 
|--|
| https://github.com/wazuh/wazuh-qa/issues/1873

### Pipeline parameters

|  Jenkins branch  | QA branch   | 
|- |- |
|  v4.2.1-rc1 |   1873-4.2.1-fim-windows-agent |

## Packages Info

| Type | Format | Architecture | Versión | Revision |  Tag | File name |
|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
| Server | rpm | x86_64 | v4.2.1 | 40214| v4.2.1 | wazuh-manager-4.2.1-0.1873.x86_64.rpm |
| Agent | rpm | x86_64 | v4.2.1 | 40214|v4.2.1| wazuh-agent-4.2.1-0.1873.x86_64.rpm |
| Agent | msi (Windows) | x86_64 | v4.2.1 | 40214| v4.2.1| wazuh-agent-4.2.1-0.1873.msi |
| Agent | pkg (macOS) | x86_64 | v4.2.1 | 40214| v4.2.1| wazuh-agent-4.2.1-0.1873.pkg |
|Agent | p5p (Solaris 11) | x86_64 | v4.2.1 | 40214|v4.2.1| wazuh-agent_v4.2.1-0.1873-sol11-i386.p5p​|
|Agent | p5p (Solaris 10) | x86_64 | v4.2.1 | 40214|v4.2.1| wazuh-agent_v4.2.1-0.1873-sol10-i386.pkg
|Agent | wpk (Linux) | x86_64 | v4.2.1 | 40214| v4.2.1 | wazuh_agent_v4.2.1-0.1873_linux_x86_64.wpk
|Agent | wpk (Windows) | x86_64 | v4.2.1 | 40214|v4.2.1| wazuh_agent_v4.2.1-0.1873_windows.wpk

**Note**: The tag is `v4.2.1` but for now, to create it on jenkins you need to use `v4.2.1-rc1`.

#### Manager
```
syscheck.debug=2
analysisd.debug=2
monitord.rotate_log=0
```

#### Agent
```
agent.debug=2
syscheck.debug=2
monitord.rotate_log=0
windows.debug=2
```

### Environment 

Provider| Box| OS| CPU| Memory | 
--|--|--|--|--|
Vagrant | "gusztavvargadr/windows-10" | Windows | 2 | 2048 |

### pytest_args
`-v --tier 0 --tier 1 --tier 2 --fim_mode="realtime" --fim_mode="whodata"`

## Development

Fix `test_fim/test_files/test_tags` to Windows. 

If the test work successfully after 2 times, you should see if other member could be verify that it woks successfully. 

## Tasks
- [x] Research about the Test Case.
- [x] Fix the Test Case.
- [x] Verify that each execution works successfully. (Full Green - Attach evidence.)
- [x] Verify that cover `scheduled`, `realtime`, and  `whodata`. Research if this case require all args, or only one.

**Comment by MizugorouZ** [comment]
These are the results of executing `test_fim/test_files/test_tags` in Windows:
| Round | OS - Manager/Agent - Module | Date | By | Reports | Notes |
|:--:|:--:|:--:|:--:|:--:|:--:|
| R1 | Windows 10 - Agent - `test_fim/test_files/test_env_variables` | 2021/10/01 | @MizugorouZ | [:red_circle:](https://github.com/wazuh/wazuh-qa/files/7268784/R1-1951.zip) | Executed without disabling any component |
| R2 | Windows 10 - Agent - `test_fim/test_files/test_env_variables` | 2021/10/01 | @MizugorouZ | [:green_circle:](https://github.com/wazuh/wazuh-qa/files/7268787/R2-1951.zip) | Components disabled manually (sca, rootcheck, syscollector) |

**Comment by damarisg** [comment]
I am attaching the result after it was merged, to verify that it continues to work.

| Results | Date | By | Status |
|:--:|:--:|:--:|:--:|
[R-1951](https://github.com/wazuh/wazuh-qa/files/7282668/R-1951.zip)  | 2021/10/05| Seyla | 🟢 | 



It closed by #1966
